code/modules/ruins/ruin_areas.dm

```diff
@@ -0,0 +1,48 @@
+//Parent types
+
+/area/ruin/
+	name = "\improper Unexplored Location"
+	icon_state = "away"
+	has_gravity = 1
+
+
+/area/ruin/unpowered
+	always_unpowered = 0
```

Comment:
I really don't know. I noticed that, but it is like that on /tg/. I'll test with that on 1 and report back.